### ISSUE TITLE
plot,adp-method spitting `mai.palette` errors for some `which=` values

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1561]
Example:
```r
library(oce)
data(adp)
plot(adp, which=19)
Warning messages:
1: In plot.window(...) : "mai.palette" is not a graphical parameter
2: In plot.xy(xy, type, ...) : "mai.palette" is not a graphical parameter
3: In title(...) : "mai.palette" is not a graphical parameter
```

**Comment by dankelley** [comment]
I've started working on this, and pushed to "develop" but this branch is *not* in a complete state yet, because I also added to the test suite a test of *every single* plot type listed in `?"plot,adp-method"`, and (at least) one of the tests is failing. I'll fix it up later this morning.

**Comment by dankelley** [comment]
@richardsc -- maybe you can check?

I think this is fixed in the "develop" branch commit 8bd03361b5b29fec68038fdff2a5eeea891c463f. I have also added tests of every single documented `which` value (see `tests/testthat/test_adp.R` and `tests/testthat/test_ad2cp.R`).

**Comment by richardsc** [comment]
Looks good. Thanks!
